lib/compat/wordpress-6.5/fonts/class-wp-font-utils.php

```diff
@@ -39,26 +69,19 @@ public static function sanitize_font_family( $font_family ) {
 				return '';
 			}
 
-			$font_family           = sanitize_text_field( $font_family );
-			$font_families         = explode( ',', $font_family );
-			$wrapped_font_families = array_map(
-				function ( $family ) {
-					$trimmed = trim( $family );
-					if ( ! empty( $trimmed ) && str_contains( $trimmed, ' ' ) && ! str_contains( $trimmed, "'" ) && ! str_contains( $trimmed, '"' ) ) {
-							return '"' . $trimmed . '"';
+			$output          = trim( sanitize_text_field( $font_family ) );
```

Comment:
`sanitize_text_field` already applies `trim`, so adding `trim` here is redundant.